src/routes/backoffice.ts

```diff
@@ -382,11 +382,11 @@ export const processQueuedBooking = async (
         },
       };
     }
-
-    // direct debits come with a negative value
-    booking.amount.value = -Math.abs(booking.amount.value);
   }
 
+  // all queued bookings should have negative amounts
+  booking.amount.value = -Math.abs(booking.amount.value);
+
```

Comment:
This was approved by Decebal

Reply:
yeah .. but do it as a rule in the UI .. not in the BE code

Reply:
I don't think the effort of doing this on the FE is worth it.

Reply:
https://github.com/etvascom/mock-solaris/pull/24/commits/40534eb2e59250ecec43dd60d5e9ba9e93203ecd